ROADMAP.md

```diff
@@ -2,13 +2,10 @@
 
 ### Short Term
 
-* Addon API and addons
-* A clear guide to hack React Storybook
-* React Native Support
+* Mobile support
 
 ### Long Term
 
 * Automatic story generation (and edge case detection) based on propTypes.
 * Angular Support
 * Vue Support
-* UI addons (Add different panels like Action Logger)
```

Comment:
> -* A clear guide to hack React Storybook

I think we need the new one since we've changed everything in the repo.

> -* Addon API and addons
-* React Native Support
-* UI addons (Add different panels like Action Logger)

Do we have any place to move it instead of just delete? Could it be cool to show wich roadmap points already done? What do you think about it, maybe right here one more section with the title 'realized steps' or checkboxes?